src/main/java/com/onevizion/scmdb/dao/DbScriptDaoOra.java

```diff
@@ -50,10 +51,18 @@ public class DbScriptDaoOra extends AbstractDaoOra {
         return dbScripts;
     };
 
-    public Map<String, SqlScript> readMap() {
+    @Cacheable("dbScriptsMap")
+    public Map<String, SqlScript> readMapCached() {
         return jdbcTemplate.query(READ_ALL, dbScriptsExtractor);
     }
 
+    public Map<String, SqlScript> readMapByIds(Set<Long> ids) {
```

Comment:
do you really need to return `Map` here? The only usage is converted to `List`

Reply:
fixed